data/_profiles/BeaconV2.yaml

```diff
@@ -2,14 +2,11 @@
 name: BeaconV2
 description: "The Beacon Project is developed under a Global Alliance for Genomics and Health (GA4GH) Iniciative for the federated discovery of genomic data in biomedical research and clinical applications. The version 2 (v2) of the Beacon protocol has been accepted as GA4GH standard in Spring 2022. MOLGENIS EMX2 features a partial implementation of Beacon v2 with a number of extensions. For more information on Beacons, see https://beacon-project.io/"
 
-models:   fairdatahub/beaconv2/molgenis.csv,
```

Comment:
huh, which models are now included. All of them?
That would be great. If we would then have tags on tables as well (not only columns) there is a very easy way to create a profile.

Reply:
Indeed, no models are specified - whatever is tagged is used! Table tags sound like a good improvement, however: if a model only uses a subset of a table, you still need to tag the 'table definition' for the table description/semantics etc. Rule could be: if table is tagged and nothing else, everything is used - if table is tagged but specific columns also, only those columns are used?